index.json

```diff
@@ -24,5 +24,8 @@
   "sargo": "Google Pixel 3a (sargo)",
   "oneplus2": "Oneplus 2",
   "pro1": "F(x)tec Pro1",
-  "lavender": "Xiaomi Redmi Note 7"
+  "lavender": "Xiaomi Redmi Note 7",
+  "trlte": "Samsung Galaxy Note 4",
+  "trltespr": "Samsung Galaxy Note 4",
+  "trltetmo": "Samsung Galaxy Note 4"
```

Comment:
This will be super confusing, add some identifying information to the human-readable names to distinguish between the three variants